ui/src/main/webapp/package.json

```diff
@@ -33,25 +33,30 @@
     "@angular/platform-browser": "^2.4.5",
     "@angular/platform-browser-dynamic": "^2.4.5",
     "@angular/router": "^3.4.5",
-    "bootstrap": "3.3.6",
-    "bootstrap-datepicker": "1.4.0",
-    "bootstrap-select": "1.7.3",
-    "bootstrap-switch": "3.3.2",
+    "angular2-moment": "^1.1.0",
```

Comment:
Please do not reorder dependencies, it makes it more difficult to see the actual changes.
(I guess this was done by npm after installing some new dependency).

We could eventually reorder it in separate PR and without updating dependencies versions.